### ISSUE TITLE
An exception occurred while establishing a connection to figure out your platform version.

### ISSUE THREAD

**Issue by kromonos** [mautic/mautic#10729]
### Mautic Version

4.0.0

### PHP version

7.4.29

### MySQL version

8.0.23

### What happened?

AH01071: Got error 'PHP message: Doctrine\\DBAL\\DBALException: An exception occurred while establishing a connection to figure out your platform version.\nYou can circumvent this by setting a 'server_version' configuration value\n\nFor further information have a look at:\nhttps://github.com/doctrine/DoctrineBundle/issues/673 - in file /var/customers/webs/kromonos/mautic.bka.li/vendor/doctrine/doctrine-bundle/ConnectionFactory.php - at line 134'

### How can we reproduce this issue?

* Install
* Use a bit
* Clear cache with `rm -rf var/cache/*`

### Relevant log output

```shell
AH01071: Got error 'PHP message: Doctrine\\DBAL\\DBALException: An exception occurred while establishing a connection to figure out your platform version.\nYou can circumvent this by setting a 'server_version' configuration value\n\nFor further information have a look at:\nhttps://github.com/doctrine/DoctrineBundle/issues/673 - in file /var/customers/webs/kromonos/mautic.bka.li/vendor/doctrine/doctrine-bundle/ConnectionFactory.php - at line 134'
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by Zrihn** [comment]
Hello! 
On my side it was caused by the php version being updated on the server but not the drivers.
So first, I had to do apt-get install php7.4-mysql to get the PDO exception removed
And then I had the following error => CurlFactory.php line 223:  Undefined constant 'CURLOPT_CUSTOMREQUEST' so I needed to execute apt-get install php7.4-curl
Then I could execute some mautic commands like fetching contacts and so on. Hope that helps you!
